src/v/kafka/latency_probe.h

```diff
@@ -77,7 +77,7 @@ class latency_probe {
               .aggregate({sm::shard_label}),
             sm::make_histogram(
               "request_latency_seconds",
-              sm::description("Internal latency of kafka consume requests"),
+              sm::description("Latency of the initial read in a kafka consume request"),
```

Comment:
Still seems pretty unclear to the uninitiated (pretty much everyone), I think an even longer description could help, maybe something like:

Latency of the only the initial read during a fetch request: this read does not block and examines all partitions involved in the fetch request for available data... (stopped because of below problem).

Reply:
@ballard26 I don't think the #HELP even appears correctly for this metric? It is the same metric name as the one right above, only the `request` labels vary (`produce` vs `fetch`), so only one #HELP line is generated for the whole family, using the text from the produce version. I don't think prom format has any way to express that the same metric name should have different HELP depending on the label value. This is a good indication that maybe the decision to have these two label values to distinguish produce/consume (as opposed to 2 names) was probably a mistake.

So I think all the HELP for both variants needs to go into the HELP for the produce variant (or better the same string should be used for both and must cover both cases).